Cargo.toml

```diff
@@ -11,8 +11,12 @@ documentation = "https://docs.rs/async-lock"
 keywords = ["lock", "mutex", "rwlock", "semaphore", "barrier"]
 categories = ["asynchronous", "concurrency"]
 
+[features]
+wasm-bindgen = ["instant", "instant/wasm-bindgen"]
+
 [dependencies]
 event-listener = "2.5.1"
+instant = { version = "0.1.9", optional = true }
```

Comment:
why does this not `target.'cfg(target_arch = "wasm32")'` dependencies?

Reply:
🤦‍♂️ Yeah, that makes sense. Then we don't need the feature. Fixed! 👨‍🔧